test/integration/puppetlabs/services/certificate_authority/certificate_authority_int_test.clj

```diff
@@ -217,8 +208,8 @@
         app
         {:jruby-puppet {:master-conf-dir master-conf-dir}}
         (let [response (http-client/put
-                        (str "https://localhost:8140"
-                            "puppet-ca/v1/certificate_status/test_cert_ca_true")
+                        (str "https://localhost:8140/"
```

Comment:
These have always been wrong, but apparently weren't failing before. Maybe the legacy routes service used to do some munging that avoided this?

Reply:
That's so weird

Reply:
Maybe due to https://github.com/puppetlabs/puppetserver/blob/f296fdf660535c48e80cf74f2e823a7b6bd848f3/src/ruby/puppetserver-lib/puppet/server/http_client.rb#L151-L153?

Reply:
Except that isn't in the new client implementation... which is where these were failing.

Reply:
That is removed from the new client implementation, but the tests still pass before this PR.